src/instructlab/model/train.py

```diff
@@ -119,13 +234,16 @@ def train(
         # By default, generate output-dir is used as train input-dir
         input_dir = ctx.obj.config.generate.output_dir
 
-    effective_data_dir = Path(data_dir or "./taxonomy_data")
+    if four_bit_quant and device != "cuda":
+        ctx.fail("--4-bit-quant option requires --device=cuda")
```

Comment:
```suggestion
        ctx.fail("'--4-bit-quant' option requires '--device=cuda'")
```

Reply:
nit

Reply:
Readability is important for users :smile:

Reply:
done.